security-questionnaire.md

```diff
@@ -0,0 +1,99 @@
+# [Security and Privacy Self-Review Questionnaire] for [Identifiers for WebRTC's Statistics API]
+
+## [3.1 Does this specification deal with personally-identifiable information?]
+
+Not beyond what is already exposed in [WebRTC 1.0].
+
+## [3.2 Does this specification deal with high-value data?]
+
+No
+
+## [3.3 Does this specification introduce new state for an origin that persists across browsing sessions?]
+
+No.
+
+## [3.4 Does this specification expose persistent, cross-origin state to the web?]
+
+Not beyond what is already exposed by the [WebRTC 1.0] specification itself
+
+## [3.5 Does this specification expose any other data to an origin that it doesn’t currently have access to?]
+
+The properties exposed by `RTCReceivedRTPStreamStats`, `RTCRemoteInboundRTPStreamStats`, `RTCSentRTPStreamStats`, `RTCOutboundRTPStreamStats`, `RTCRemoteOutboundRTPStreamStats`, `RTCIceCandidatePairStats`, `RTCTransportStats` expose network-layer data not currently available to the JavaScript layer.
+
+In addition, the behavior of WebRTC Stats (in particular `RTCMediaStreamTrackStats`) on [isolated media streams] (which aren't otherwise available to the JavaScript layer) is currently ill-defined.
```

Comment:
Can you clarify what you mean by "the behavior of WebRTC Stats (...) is currently ill-defined"?

Reply:
it's not clear how much statistics should be reported on isolated media streams given that these statistics leak information about their content - see https://github.com/w3c/webrtc-pc/issues/1613

Reply:
I think the current behavior is perfectly defined (the same stats are available as for other tracks), but there's a good chance that this is not what we want.
My suggestion: s/is currently ill-defined/is a topic that needs further discussion, since the current lack of special treatment for these streams may raise issues/.

Reply:
I corrected this into "we expect that the behavior of WebRTC Stats (in particular `RTCMediaStreamTrackStats`) on [isolated media streams] (which aren't otherwise available to the JavaScript layer) will need further work (see https://github.com/w3c/webrtc-pc/issues/1613)"